v3.0/how-to/scale/with-ansible.md

```diff
@@ -207,6 +207,10 @@ For example, if you want to add a PD node (node103) with the IP address `172.16.
         {deploy_dir}/scripts/start_pd.sh
         ```
 
+        > **Note:**
+        >
+        > You need to confirm that the `health` status of the newly added PD node is "true" using [PD Control](/v3.0/reference/tools/pd-control.md). Otherwise, the PD service fails to start and an error message `["join meet error"] [error="etcdserver: unhealthy cluster"]` is returned in the log.
```

Comment:
```suggestion
        > Before start, you need to ensure that the `health` status of the newly added PD node is "true", using [PD Control](/v3.0/reference/tools/pd-control.md). Otherwise, the PD service might fail to start and an error message `["join meet error"] [error="etcdserver: unhealthy cluster"]` is returned in the log.
```

Reply:
"ensure" is stronger than "confirm".  And "otherwise" only suggests a possibility, not a definite thing.